CHANGES-5.0.md

```diff
@@ -7,7 +7,8 @@
 
 ## Enhancements
 
-* change the `/gateway` API path to plural form. [#8823](https://github.com/emqx/emqx/pull/8823)
+* Change the `/gateway` API path to plural form. [#8823](https://github.com/emqx/emqx/pull/8823)
+* Remove `node.etc_dir` from emqx.conf. [#8892](https://github.com/emqx/emqx/pull/8892)
```

Comment:
```suggestion
* Remove `node.etc_dir` from emqx.conf, because it is never used.
  Also allow user to customize the logging directory [#8892](https://github.com/emqx/emqx/pull/8892)
```